Source/Scripts/SourceToBinary.cs

```diff
@@ -216,7 +216,11 @@ private void SerializeObject(object obj) {
 
       var fieldName = f.Name;
       if (fieldName.StartsWith("<") && fieldName.EndsWith("k__BackingField")) {
+        // Support auto properties
         fieldName = fieldName.Substring(1, fieldName.IndexOf(">", StringComparison.Ordinal) - 1);
+      } else if (fieldName.StartsWith("<") && fieldName.EndsWith(">P")) {
```

Comment:
Do you have an example of what such a field would look like?

Reply:
`<endToken>P`, created because of `public class TokenRange(Token startToken, Token? endToken)`, is the name of the field for which this was added.